source/validate.h

```diff
@@ -190,4 +190,11 @@ spv_result_t spvValidateIDs(const spv_instruction_t* pInstructions,
                             spv_position position,
                             const spvtools::MessageConsumer& consumer);
 
+// Performs validation for the SPIRV-V module binary and returns the
+// ValidationState to the caller.
+spv_result_t ValidateAndReturnValidationState(
```

Comment:
This should be inside the spvtools namespace.

All the public names in the C API start with "spv". We want to keep that pattern here as well.  So with a name that doesn't start with "spv" you should put it inside the spvtools namespace.